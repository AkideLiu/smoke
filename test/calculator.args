fixtures/calculator.rb
fixtures/calculator
