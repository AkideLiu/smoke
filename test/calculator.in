--separated-args fixtures/calculator fixtures/calculator.rb
