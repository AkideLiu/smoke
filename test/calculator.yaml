tests:
  calculator:
    args:
      - fixtures/calculator
    exit-status: 1
    stdout:
      file: io/calculator.out
