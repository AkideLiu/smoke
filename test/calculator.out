addition
[32m  succeeded[0m
division
[32m  succeeded[0m
multiple-digits
[32m  succeeded[0m
multiplication
[32m  succeeded[0m
subtraction
[32m  succeeded[0m

[32m5 tests, 0 failures[0m
